docs/rules/template-indent.md

```diff
@@ -0,0 +1,66 @@
+# ember/template-indent
+
+🔧 This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).
+
+<!-- end auto-generated rule header -->
+
+This rule extends the base [eslint indent](https://eslint.org/docs/latest/rules/indent) rule, but only applies the indents to Glimmer Nodes.
+
+Otherwise, it receives the same options as the original and can run together with the base rule.
+
+## Configuration
+
+<!-- begin auto-generated rule options list -->
+
+| Name             | Type     | Default |
+| :--------------- | :------- | :------ |
+| `ignoreComments` | Boolean  | `false` |
+| `ignoredNodes`   | String[] |         |
+
+<!-- end auto-generated rule options list -->
+
+## Rule Details
+
+Enforce consistent indentation for fcct templates
+
+```js
+const rules = {
+  'ember/template-indent': [
+    'error',
+    2, // or 'tab'
+    {
+      ignoreComments: false,
+      ignoredNodes: []
+    }
+  ]
+};
+```
```

Comment:
Let's omit this. We don't need a basic example of how to configure a rule.

```suggestion
```

If you want to add more details about each rule option, add a `description` property to the options in the schema and re-run `yarn update:eslint-docs`.